editor/src/clj/editor/atlas.clj

```diff
@@ -23,7 +23,7 @@
             [editor.gl.pass :as pass]
             [editor.gl.shader :as shader]
             [editor.gl.texture :as texture]
-            [editor.gl.vertex :as vtx]
+            [editor.gl.vertex2 :as vtx]
```

Comment:
A note that @Jhonnyg is the author of the rendering changes for the editor.
(to correctly display the sprite geometry that an atlas may have)